src/world.js

```diff
@@ -34,6 +34,34 @@
 
 (function( $ ){
 
+// For async debugging to avoid console forced synchronization,
+// use this buffered logger.
+var _logs = [];
+setInterval(() => {
```

Comment:
I'd rather we not have this setInterval always running. Instead we should have this log utility be turned off by default but easy to turn on if you need it for debugging.